.github/workflows/deploy-market.yaml

```diff
@@ -9,6 +9,7 @@ on:
           - fuji
           - kovan
           - mainnet
+          - goerli
```

Comment:
You might to also add an entry for `goerli` in line 33 (`ethereum_url` field). Same for the other workflows.

Reply:
good catch.

that line is going to be awful when we've got 5-6 deployments (with their corresponding testnets)

Reply:
updated